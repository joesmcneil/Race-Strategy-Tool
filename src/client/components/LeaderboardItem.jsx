const LeaderboardItem = (props) => {
  // This component just returns some div elements with some inserted racer props
  return (
      <div className="child">
        <div className="parent">
          <div className="child">
            {props.position}
          </div>
          <div className="child">
            {props.alias}
          </div>
          <div className="child">
            {props.timeDelta}
          </div>
        </div>
      </div>
  );
};

export default LeaderboardItem;
