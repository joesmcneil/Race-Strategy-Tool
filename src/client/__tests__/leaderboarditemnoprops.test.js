import LeaderboardItem from "../components/LeaderboardItem";
import renderer from 'react-test-renderer';

// Testing the LeaderboardItem component renders without being passed props

it('should render without being passed any props', () => {
  const expectedComponent = renderer.create(
  <LeaderboardItem />
  ).toJSON();

  expect(expectedComponent).toMatchSnapshot()
})
