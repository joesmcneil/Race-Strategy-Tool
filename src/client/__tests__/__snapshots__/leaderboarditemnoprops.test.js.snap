// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render without being passed any props 1`] = `
<div
  className="child"
>
  <div
    className="parent"
  >
    <div
      className="child"
    />
    <div
      className="child"
    />
    <div
      className="child"
    />
  </div>
</div>
`;
